vscode-dotnet-sdk-extension/package.json

```diff
@@ -72,7 +72,7 @@
 		}
 	},
 	"scripts": {
-		"vscode:prepublish": "npm run compile-all && npm install && webpack --mode production",
+		"vscode:prepublish": "npm run compile-all && npm install && dotnet build ../msbuild/signJs  --property jsOutputPath=..\\..\\vscode-dotnet-sdk-extension\\dist -v:normal && webpack --mode production  && dotnet build ../msbuild/signJs  --property jsOutputPath=..\\..\\vscode-dotnet-sdk-extension\\dist -v:normal ",
```

Comment:
This double signing is not necessary, it was more of a test to see if it matters if its done before or after the .VSIX is packed. I would like to keep it in, in case the runtime signing ever breaks in the future but this one does not due to some change in the packaging approach, then this will still work as an example.